src/Symfony/Component/Scheduler/Bridge/Microsoft/Task/TaskState.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Scheduler\Bridge\Microsoft\Task;
+
+/**
+ * @author Guillaume Loulier <contact@guillaumeloulier.fr>
+ */
+final class TaskState
+{
+    public const COMPLETED = 'Completed';
+    public const DISABLED = 'Disabled';
+    public const ENABLED = 'Enabled';
+    public const FAULTED = 'Faulted';
+}
```

Comment:
maybe add a private construct? or more those const inside an interface?

Reply:
Moved to `TaskInterface`